batcher/aligned-batcher/src/lib.rs

```diff
@@ -1646,14 +1651,30 @@ impl Batcher {
         )
         .await
         {
-            Ok(_) => info!("createNewTask transaction successfully canceled"),
+            Ok(receipt) => {
+                info!("createNewTask transaction successfully canceled");
+                let gas_cost =
+                    Self::calculate_gas_cost(receipt.effective_gas_price, receipt.gas_used);
+                self.metrics
+                    .batcher_gas_cost_cancel_task_total
+                    .inc_by(gas_cost.as_u64());
+            }
             Err(e) => error!("Could not cancel createNewTask transaction: {e}"),
         };
         self.metrics
             .cancel_create_new_task_duration
             .set(start.elapsed().as_millis() as i64);
     }
 
+    fn calculate_gas_cost(gas_price: Option<U256>, gas_used: Option<U256>) -> U256 {
```

Comment:
I would name it `calculate_gas_cost_in_gwei` or `gas_cost_in_gwei`

Reply:
Addressed in [#6c40b9a](https://github.com/yetanotherco/aligned_layer/pull/1663/commits/6c40b9af638dbc80f05e7ec73f19d4d325b256dd)!